pygmt/datasets/earth_age.py

```diff
@@ -26,6 +26,13 @@ def load_earth_age(resolution="01d", region=None, registration=None):
     registration type (**p** for pixel registration or **g** for gridline
     registration).
 
+    The default color palette table (CPT) for this dataset is *@earth_age.cpt*.
+    It's implicitly used when passing in the file name of the dataset to any
+    grid plotting function if no CPT is explicitly specified. When the dataset
```

Comment:
As they belong to the `pygmt.Figure` class, "method" is more precise than "function".
```suggestion
    grid plotting method if no CPT is explicitly specified. When the dataset
```